.github/workflows/label.yml

```
@@ -0,0 +1,19 @@
+---
+
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler/blob/master/README.md
+
+name: "Pull Request Labeler"
+on: [pull_request_target]
+
```

```suggestion
permissions: {}

```

GitHub action permissions has evolved since, and the newest best practice is to specify at the workflow level the lowest permissions needed, and at the job level the minimum permissions needed. These permissions declare what the `GITHUB_TOKEN` is granted to do.


The advantage of setting the least at the workflow level is that further changes to the workflow by adding a job doesn't unintentionally give more rights that was previously planned out, but overlooked afterwards.


Declaring the permissions are quite easier to understand the config when looking at the workflow code, instead of a web interface that changes over time.